libraries/networking/src/ResourceRequest.cpp

```diff
@@ -15,8 +15,14 @@
 #include <DependencyManager.h>
 #include <StatTracker.h>
 
+#include <QtCore/QDateTime>
 #include <QtCore/QThread>
 
+QString ResourceRequest::toHttpDateString(uint64_t msecsSinceEpoch) {
+    return QLocale::c()
+        .toString(QDateTime::fromMSecsSinceEpoch(msecsSinceEpoch), QLatin1String("ddd, dd MMM yyyy hh:mm:ss 'GMT'"))
+        .toLatin1();
```

Comment:
```suggestion
    return QDateTime::fromMSecsSinceEpoch(msecsSinceEpoch)
        .toString("ddd, dd MMM yyyy hh:mm:ss 'GMT'")
        .toLatin1();
```
Would something like this be simpler?